rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -0,0 +1,67 @@
+from rj_msgs import msg
+import stp.rc as rc
+import numpy as np
+
+from typing import List
+
+def robotstate_to_robot(robot_msg: msg._robot_state.RobotState, index: int) -> rc.Robot:
+    """
+        :return: robot class representing the state of the robot.
+    """
+    robot_id = index
+
+    x = robot_msg.pose.position.x
+    y = robot_msg.pose.position.y
+    theta = robot_msg.pose.heading
+    pose = np.array([x,y,theta])
+
+    dx = robot_msg.velocity.linear.x
+    dy = robot_msg.velocity.linear.y
+    dtheta = robot_msg.velocity.angular
+    twist = np.array([dx,dy,dtheta])
+
+    has_ball: bool
+    has_ball = None
+
+    robot = rc.Robot(robot_id, pose, twist, has_ball)
+
+    return robot
+
+def ballstate_to_ball(ball_msg: msg._ball_state.BallState) -> rc.Ball:
```

Comment:
This stuff as well, I'll have to add more information to most of the converters.